api/models.py

```diff
@@ -1,29 +1,145 @@
-    community = models.TextField(max_length=100)
-    city = models.TextField(max_length=100)
-    district = models.TextField(max_length=100)
+    updated = models.DateTimeField(auto_now=True, editable=False)
+
+    class Meta:
+        abstract = True
+
+
+class MeteringBase(models.Model):
+    """
+    Base Model for Metering and MeteringHistory models.
+    """
+
+    # we do not need id field
+    # http://stackoverflow.com/questions/4814167/storing-time-series-data-relational-or-non/4884384#4884384
+    created = models.DateTimeField(auto_now_add=True, primary_key=True)
+
+    # Data Fields
+    pm01 = models.FloatField(
+        help_text='PM 0.1 in ug/m^3'
+    )
+    pm25 = models.FloatField(
+        help_text='PM 2.5 in ug/m^3'
+    )
+    pm10 = models.FloatField(
+        help_text='PM 10 in ug/m^3'
+    )
+    temp_out1 = models.FloatField(
+        help_text='Outside temperature sensor1, in C.'
+    )
+    temp_out2 = models.FloatField(
+        help_text='Outside temperature sensor2, optional, in C.',
+        blank=True,
+        default=None,
+        null=True
+    )
+    temp_out3 = models.FloatField(
+        help_text='Outside temperature sensor3, optional, in C.',
+        blank=True,
+        default=None,
+        null=True
+    )
+    temp_int_air1 = models.FloatField(
+        help_text='Internal temperature sensor1 (air sucked by PM sensor), in C.'
+    )
+    hum_out1 = models.FloatField(
+        help_text='Outside relative humidity sensor1, in %.'
+    )
+    hum_out2 = models.FloatField(
+        help_text='Outside relative humidity sensor2, optional, in %.',
+        blank=True,
+        default=None,
+        null=True
+    )
+    hum_out3 = models.FloatField(
+        help_text='Outside relative humidity sensor3, optional, in %.',
+        blank=True,
+        default=None,
+        null=True
+    )
+    hum_int_air1 = models.FloatField(
+        help_text='Internal relative humidity sensor1 (air sucked by PM sensor), in %.'
+    )
+    rssi = models.FloatField(
+        help_text='RSSI of WiFi (signal strength). For debugging "vanishing" stations.'
+    )
+    bpress_out1 = models.FloatField(
+        help_text='Outside absolute barometric pressure sensor1, in hPa.'
+    )
 
     # Relationship Fields
-    owner = models.ForeignKey(settings.AUTH_USER_MODEL, )
-    project = models.ForeignKey('api.Project', )
+    station = models.ForeignKey('api.Station')
+
+    class Meta:
+        abstract = True
+
+    def __unicode__(self):
+        return u'%s' % self.created
+
+
+class LocationBase(models.Model):
+    """
+    Base Model for location features.
+    """
+
+    position = gis_models.PointField(help_text='Exact position on map.', default=None, null=True)
+    country = models.CharField(max_length=255, default='')
+    state = models.CharField(max_length=255, default='')
+    county = models.CharField(max_length=255, default='')
+    community = models.CharField(max_length=255, default='')
```

Comment:
@zen what is rationale behind community field? Other fields are quite self explanatory...